example-app/src/reducers/reducer.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { selectIsPending } from 'redux-pending-effects';
+import SagaTester from 'redux-saga-tester';
+
+import { store } from '../store';
+import {
+  getPatents,
+  getLibraryContent,
+  getAstronomyPictureData
+} from '../actions';
+import { astronomyPictureActionNames, clearStoreActionName } from '../constants/actionNames';
+import { nasaService } from '../services/NasaService';
+import { astronomyPictureWorker } from '../sagas/astronomyPictureSagas';
+import { rootReducerWrapper } from './rootReducer';
+
+describe('store testing on RPE selector', () => {
+  afterEach(() => {
+    store.dispatch({
+      type: clearStoreActionName
+    });
+  });
+
+  it('should test selectIsPending changing on getPatents fetch', async () => {

```

Comment:
I suggest to do almost the same but twice.

`should have default negative pending state` (where you just test that default selector value from getState is `false`)

separate test for `should trigger pending state` (where you trigger action and expect the result of `true`)

and another separate test for `should complete pending state` (where you trigger action for true, and smth for its completion) and then expect the result of `false`)

Reply:
well, in this style and test the rest of the cases further
we just decomposed this test into three separate ones to better track the behavior